spring-xd-yarn/site/config/servers.yml

```diff
@@ -63,6 +72,9 @@ spring:
 #  hadoop:
 #    fsUri: hdfs://localhost:8020
 #    resourceManagerHost: localhost
+#    resourceManagerPort: 8032
+#    resourceManagerSchedulerAddress: ${spring.hadoop.resourceManagerHost:8030}
+#    jobHistoryAddress: ${spring.hadoop.resourceManagerHost:10020}
```

Comment:
It's not used, it is needed for custom batch jobs so I though we should include it here